Npm start failing on windows

```
PS C:\Data\Projects\blog> npm start

> hylia@0.4.3 start C:\Data\Projects\blog
> concurrently 'npm run sass:process -- --watch' 'npm run cms:bundle -- --watch' 'chokidar "src/_includes/**" -c "npm run cms:precompile"' 'npm run serve'

[0] 'np' is not recognized as an internal or external command,
[0] operable program or batch file.
[1] 'run' is not recognized as an internal or external command,
[1] operable program or batch file.
[2] The filename, directory name, or volume label syntax is incorrect.
[3] '--watch'' is not recognized as an internal or external command,
[3] operable program or batch file.
[4] 'np' is not recognized as an internal or external command,
[4] operable program or batch file.
[5] 'run' is not recognized as an internal or external command,
[5] operable program or batch file.
[6] The filename, directory name, or volume label syntax is incorrect.
[7] '--' is not recognized as an internal or external command,
[7] operable program or batch file.
[8] '--watch'' is not recognized as an internal or external command,
[8] operable program or batch file.
[9] 'chokida' is not recognized as an internal or external command,
[9] operable program or batch file.
[10] 'src' is not recognized as an internal or external command,
[10] operable program or batch file.
[11] '-c' is not recognized as an internal or external command,
[11] operable program or batch file.
[13] 'np' is not recognized as an internal or external command,
[13] operable program or batch file.
[14] 'run' is not recognized as an internal or external command,
[14] operable program or batch file.
[15] 'serve'' is not recognized as an internal or external command,
[15] operable program or batch file.
[14] run exited with code 1
[13] np exited with code 1
[11] -c exited with code 1
[10] src/_includes/** exited with code 1
[9] chokida exited with code 1
[8] --watch' exited with code 1
[7] -- exited with code 1
[6] cms:bundle exited with code 1
[5] run exited with code 1
[4] np exited with code 1
[3] --watch' exited with code 1
[2] sass:process exited with code 1
[1] run exited with code 1
[0] np exited with code 1
[15] serve' exited with code 1
[12] npm ERR! missing script: cms:precompile'
[12] npm ERR!
[12] npm ERR! Did you mean this?
[12] npm ERR!     cms:precompile
[12]
[12] npm ERR! A complete log of this run can be found in:
[12] npm ERR!     C:\Users\shree\AppData\Roaming\npm-cache\_logs\2019-08-15T00_48_08_034Z-debug.log
[12] npm run cms:precompile' exited with code 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hylia@0.4.3 start: `concurrently 'npm run sass:process -- --watch' 'npm run cms:bundle -- --watch' 'chokidar "src/_includes/**" -c "npm run cms:precompile"' 'npm run serve'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hylia@0.4.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\shree\AppData\Roaming\npm-cache\_logs\2019-08-15T00_48_08_084Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -31,9 +31,9 @@
   "scripts": {
     "sass:tokens": "npx json-to-scss src/_data/tokens.json src/scss/_tokens.scss",
     "sass:process": "npm run sass:tokens && sass src/scss/global.scss src/_includes/assets/css/global.css --style=compressed",
-    "cms:precompile": "make-dir dist/admin && nunjucks-precompile src/_includes > dist/admin/templates.js -i '\\.(njk|css|svg)$'",
+    "cms:precompile": "make-dir dist/admin && nunjucks-precompile src/_includes > dist/admin/templates.js -i \"\\.(njk|css|svg)$\"",
     "cms:bundle": "rollup --config",
-    "start": "concurrently 'npm run sass:process -- --watch' 'npm run cms:bundle -- --watch' 'chokidar \"src/_includes/**\" -c \"npm run cms:precompile\"' 'npm run serve'",
+    "start": "concurrently \"npm run sass:process -- --watch\" \"npm run cms:bundle -- --watch\" \"chokidar \\\"src/_includes/**\\\" -c \\\"npm run cms:precompile\\\"\" \"npm run serve\"",
     "serve": "cross-env ELEVENTY_ENV=development npx eleventy --serve",
     "production": "npm run sass:process && npm run cms:precompile && npm run cms:bundle && npx eleventy"
   },

